src/Illuminate/Database/Schema/SQLiteBuilder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Illuminate\Database\Schema;
+
+class SQLiteBuilder extends Builder
+{
+    /**
+     * Drop all tables from the database.
+     *
+     * @return void
+     */
+    public function dropAllTables()
+    {
+        $dbPath = $this->connection->getConfig('database');
+
+        if (file_exists($dbPath)) {
+            unlink($dbPath);
```

Comment:
@casperboone - thanks. I'm not worried about in-memory calls - I dont think that will ever happen.

But I didnt like that the command was doing a file manipulation - I didnt feel very right.

Using your code as a starting point - I've come up with a pure sql solution that should clear a sqlite database without actually deleting the file. Thanks.